### ISSUE TITLE
Miktex Only Installing One Package at a Time

### ISSUE THREAD

**Issue by Qutrit** [MiKTeX/miktex#167]
I have a fresh install of Miktex version 2.9.6751. I am using TeXworks and Texmaker for the front end. I have a tex file that uses a lot of packages. I'm on a Windows 10 machine.

Whenever I try to compile (typeset?) the latex, it would prompt me to install a dependency, but it only installs that one dependency and then it says the next dependency is not found. The only way I can get it to continue, is to stop and then start the compiling again. Then it installs the next dependency and it says it can't find the following one. I have to keep doing this for all the dependencies. I've attached the log snippet (note: I removed username). Thanks.

[pdflatex.log](https://github.com/MiKTeX/miktex/files/2207249/pdflatex.log)

**Comment by Qutrit** [comment]
I figured out a work around. Open miktex console and select restart as admin. After miktex restarts, go to settings and under general select always install missing packages on-the-fly. Don't close miktex console just minimize it.

**Comment by Qutrit** [comment]
I don't why this has been closed, but this is a bug that needs to be fixed. Maybe the information was unclear. Miktex installs only 1 package and then FAILS with error 87. The only way to make it do another package is to start the compiling again, but each time it only installs 1 package and then FAILS with error code 87. Eventually you get the document to compile after restarting the compile a bunch of times. I tried TeXworks and Texmaker and both had the same issue so I think the problem is with Miktex.
